pkg/agent/openflow/pipeline.go

```diff
@@ -1061,6 +1086,9 @@ func (c *client) traceflowL2ForwardOutputFlows(dataplaneTag uint8, liveTraffic,
 			MatchRegRange(int(marksReg), portFoundMark, ofPortMarkRange).
 			Cookie(c.cookieAllocator.Request(category).Raw())
 		if !droppedOnly {
+			if c.ovsMetersAreSupported {
+				fb = fb.Action().Meter(PacketInMeterIDTF)
```

Comment:
Maybe also add Meter for `fbHairpin` below. L1109

Reply:
Done, I should have merged this PR earlier :)